tests/models/test_implicit_als.py

```diff
@@ -346,3 +346,63 @@ def test_i2i_with_warm_and_cold_items(self, use_gpu: bool, dataset: Dataset) ->
                 dataset=dataset,
                 k=2,
             )
+
+    def test_fit_partial(self, use_gpu: bool, dataset: Dataset) -> None:
+        base_model = AlternatingLeastSquares(factors=8, num_threads=2, use_gpu=use_gpu, random_state=1)
+        model = ImplicitALSWrapperModel(model=base_model).fit(dataset)
+        data = [
+            [150, 11],
+            [150, 12],
+            [150, 15],
+        ]
+        new_interactions = pd.DataFrame(data, columns=Columns.UserItem)
+        new_interactions[Columns.Weight] = 1
+        new_interactions[Columns.Datetime] = "2021-09-10"
+        new_dataset = dataset.construct_new_datasets(new_interactions)
+        model.fit_partial(new_dataset)
+        actual = model.recommend(
+            users=[150],  # new user
```

Comment:
Could you please request here recomendation for one user from original dataset as well? To ensure we didn't lose its data (same for lightfm)

Reply:
Thanks, fixed ea33c05